### PR TITLE
testing travis one level up

### PR DESCRIPTION
**Pull request by y3l2n**

### CODE REVIEW

**Comment by y3l2n** [comment]
Making the migration to monorepo in smaller, non-breaking stages.

First step - make sure travis works and build works when a single package is one level down.

File: package.json [hunk 1]
```diff
@@ -1,86 +1,4 @@
...
-    "@types/rimraf": "^2.0.2",
-    "@types/terminal-link": "^1.1.0",
-    "@types/uuid": "^3.4.5",
-    "jest": "^24.8.0",
-    "prettier": "^1.18.2",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.18.0",
-    "typescript": "^3.5.3"
-  }
+  "name": "data-studio-developer-tools",
```
Comment (diminishedprime) [inline package.json]: If we don't have any top level dependencies, we should just omit this file.

File: .travis.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ branches:
 cache:
   directories:
   - node_modules/
-script:  yarn travis
+script: bash test.sh
```
Comment (diminishedprime) [inline .travis.yml]: As long as test.sh has a shebang (`#!/bin/bash`), we don't need to invoke this with bash.

File: publish-to-npm.sh [hunk 3]
```diff
@@ -23,6 +23,8 @@ fi
 
 set -x
 
+# Go to the package directory
```
Comment (diminishedprime) [inline publish-to-npm.sh]: This script should probably still live in the dscc-gen directory. The top-level publish command should then delegate to each sub-package's custom publish script.

File: test.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd packages/dscc-gen
```
Comment (diminishedprime) [inline test.sh]: Same comment as above.

File: package.json (hunk 1 above)
Comment (y3l2n) [reply package.json]: doing that for now

File: publish-to-npm.sh (hunk 3 above)
Comment (y3l2n) [reply publish-to-npm.sh]: I agree- done.

File: test.sh (hunk 4 above)
Comment (y3l2n) [reply test.sh]: I agree

**Pull request merged**